incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+# Incident Response Plan
+
+## Purpose  
+This document outlines the process for handling security-impacting incidents affecting one or more projects in the Foundation’s ecosystem. Incidents may include platform changes with security implications, account compromise, or other security events that require coordinated response.
+
+The Foundation’s role is to:  
+1. **Receive and triage THE REPORTS**  
+2. **Connect reporters and affected maintainers with the right experts**  
+3. **Facilitate coordinated response** across multiple projects when needed  
+4. **Communicate clearly and act as the contact point** while respecting confidentiality and responsible disclosure principles  
+
+---
+
+## Scope  
+
+This plan covers incidents such as:  
+- **Platform changes or providers outages** (e.g., GitHub UI change) that create security or operational risk.  
+- **Account or registry access issues** (e.g., npm lockdown, compromised maintainer account).  
+- **Supply chain attacks** (e.g. XZ, phishing campaign, etc.. )  
+- **Third-party service compromises** affecting Foundation projects (e.g., data leaks in external services)  
```

Comment:
I think this is probably a dupe of the platform change one if my changes are accepted.